doc/source/conf.py

```diff
@@ -341,31 +391,6 @@ def build_versions_table(branch: str) -> list[str]:
     return table
 
 
-def get_release_branches_in_metapackage():
-    """Retrieve the release branches in the PyAnsys metapackage."""
-    # Get the PyAnsys metapackage repository
```

Comment:
Wouldn't it be good to still use this for the ``supported_python_versions_by_metapackage_version `` dict? Otherwise, we will have to manually update it I guess :S

Reply:
That's a good point. Thanks for raising it. Let me restore this this function and adapt it to the Jinja context.

Reply:
Fixed in https://github.com/ansys/pyansys/pull/830/commits/46bf0781e6b0b5c3609adbc97f6f5fc773cc7a46